Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Set up basic python environment:
+FROM phusion/baseimage:latest
```

Comment:
Given that all these packages need to run as part of edxapp, we really should have a standard Docker container with everything set up.

Reply:
This is to run `xblock-sdk`, not `edxapp`. And there is [a standard container for xblock-sdk](https://hub.docker.com/r/jbarciauskas/xblock-sdk/) kind of, but it's two years out of date and so wasn't usable with newer package versions.